action.yml

```diff
@@ -2,8 +2,8 @@ name: 'Publish to npm'
 description: 'Publish the release to npm'
 inputs:
   npm-token:
-    description: 'The token used for npm publishing. Publishing to npm will be skipped if omitted.'
-    required: true
+    description: 'The token used for npm publishing. If omitted the action will perform a --dry-run publish.'
```

Comment:
[nit] I'm not sure everyone will know (or care) about what this action is doing under the hood, so I'm wondering if it's slightly friendlier to merely say "the action will perform a dry run".